### ISSUE TITLE
Themes - Feedback and ideas for Breeze Dark and Zephyr Dark.

### ISSUE THREAD

**Issue by technatica** [leo-editor/leo-editor#1219]
Please use this issue for feedback on the two themes and ideas for the future.

My goal is to give Leo a look that is functional and easy on the eyes.

**Comment by maphew** [comment]
I confirm 360d9762 fixes the blue box outline on Win10. Thanks!

Next problem: Render pane text is black and hard to read.
https://i.imgur.com/orRxHIh.png

**Comment by technatica** [comment]
I will see if the render pane is amenable to styling.

Chris

**Comment by maphew** [comment]
How to set the base font size?

I found `ZephyrDarkTheme.leo#@settings-->@theme Zephyr Dark-->@data qt-gui-plugin-style-sheet-->Universal Style` and changed it to 10px from 12px. After restarting the only text that was visibly changed was the Render pane.

**Comment by technatica** [comment]
I cannot change the individual style of the QTextBrowser component that is
a child of ViewRenederedController.

I changed the color of the Universal style to make the text white. It seems
that the rendered pane is about the last item that the Universal Style
applies to. I imagine there are a few plugins that also respond to it as I
likely haven't styled them specifically.

I use gammaray to ID the objects but in this case nothing I do sticks.

Updated version coming up.

Chris
